printers/initial-setup.cfg

```diff
@@ -59,6 +59,16 @@ command: /home/pi/printer_data/config/RatOS/scripts/install-prusa-mini-config.sh
 timeout: 2
 verbose: False
 
+[gcode_shell_command copy_prusa_mk3s]
+command: /home/pi/klipper_config/config/scripts/install-prusa-mk3s-config.sh
```

Comment:
change `klipper_config` to `printer_data/config`

Reply:
Done